<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyGoWave Desktop Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Wave Inbox</string>
          </property>
          <property name="indent">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="waveList"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actConnect"/>
    <addaction name="actDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actNew"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;?</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Preferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actConnect"/>
   <addaction name="actDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actNew"/>
  </widget>
  <action name="actNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../main.qrc">
     <normaloff>:/gui/icons/document-new.png</normaloff>:/gui/icons/document-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="actConnect">
   <property name="icon">
    <iconset resource="../main.qrc">
     <normaloff>:/gui/icons/network-connect.png</normaloff>:/gui/icons/network-connect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="../main.qrc">
     <normaloff>:/gui/icons/application-exit.png</normaloff>:/gui/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../main.qrc">
     <normaloff>:/gui/icons/network-disconnect.png</normaloff>:/gui/icons/network-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actLeaveWave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../main.qrc">
     <normaloff>:/gui/icons/system-shutdown.png</normaloff>:/gui/icons/system-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Leave Wave</string>
   </property>
  </action>
  <action name="actOpenWave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../main.qrc">
     <normaloff>:/gui/icons/document-open.png</normaloff>:/gui/icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Wave</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="icon">
    <iconset resource="../main.qrc">
     <normaloff>:/gui/icons/configure.png</normaloff>:/gui/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
